docs/4.0/docs/cli/apply.md

```diff
@@ -4,9 +4,46 @@
 $ sealos apply -f Clusterfile
 ```
 
-## Experimental
+Example Clusterfile:
 
-`Clusterfile` can also be written using go-template syntax(just like [helm](https://helm.sh/)! but some template functions are missing, like `include`/`tpl`/`require`/`lookup`). for example, we create a file named `Clusterfile.yaml`
+```yaml
+apiVersion: apps.sealos.io/v1beta1
+kind: Cluster
+metadata:
+  name: default
+spec:
+  # 服务器 IP 地址列表和角色

```

Comment:
Chinese here needs translation?

Reply:
Sorry I forgot. Will add this in the next PR.